internal/buildscripts/packaging/choco/splunk-otel-collector/tools/common.ps1

```diff
@@ -0,0 +1,100 @@
+try {
+    $drive = (Get-ToolsLocation | Split-Path -Qualifier)

```

Comment:
Not needed anymore with the suggested changes below.

Reply:
Removed